web/tools.html

```diff
@@ -105,134 +105,15 @@ <h4>Tools</h4>
-                                manifestLabelField.innerHTML = manifestLabel + "&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;" + manifestType + "&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;" + manifestItemCount;
-                            }
-                            catch{
-                                loadMessage.innerHTML = "No metadata available!";
-                                manifestLabelField.innerHTML = "";
-                            }
-                        })
-                        .catch(error => {
-                            manifestMessage.textContent = 'Failed to load manifest. Please check the URL and try again.';
-                            manifestMessage.style.color = 'red';
-                            console.error('Error:', error);
-                        });
-                });
-            });
-        </script>
-        <script>
-            function toggleDropdown() {
-                const manifestContainer = document.getElementById('stored_manifest_links');
-                const dropdownArrow = document.getElementById('dropdownArrow');
-
-                if (manifestContainer.style.display === 'none' || manifestContainer.style.display === '') {
-                    manifestContainer.style.display = 'block';  // Show content
-                    dropdownArrow.textContent = '▲';  // Change arrow to up
-                } else {
-                    manifestContainer.style.display = 'none';   // Hide content
-                    dropdownArrow.textContent = '▼';  // Change arrow to down
-                }
-            }
-
-            // Add click event to both label and arrow to toggle the dropdown
-            document.getElementById('dropdownLabel').addEventListener('click', toggleDropdown);
-            document.getElementById('dropdownArrow').addEventListener('click', toggleDropdown);
-
-            function renderStoredManifests() {
-                const manifestContainer = document.getElementById('stored_manifest_links');
-                const storedManifests = getStoredManifestLinks();
-
-                if (!manifestContainer) {
-                    console.error("Manifest container not found.");
-                    return;
-                }
-
-                manifestContainer.innerHTML = '';  // Clear previous content
-
-                if (storedManifests.length === 0) {
-                    manifestContainer.innerHTML = '<p>No stored manifest links.</p>';
-                    return;
-                }
-
-                storedManifests.forEach(manifestLink => {
-                    const manifestHTML = `
-                        <a href="${manifestLink}" target="_blank" class="manifestLink">
-                            <p>${manifestLink}</p>
-                        </a>
-                    `;
-                    manifestContainer.innerHTML += manifestHTML;
-                });
-            }
-
-        </script>
     </body>
 </html>
```

Comment:
So far so good, the comments are very informative as to what your code is doing. Great work!